docs/source/federation-2/moving-to-federation-2.mdx

```diff
@@ -122,11 +122,25 @@ First, add the following definition to your subgraph schema:
 
 ```graphql
 extend schema
-    @link(url: "https://specs.apollo.dev/federation/v2.0",
-          import: ["@key", "@requires", "@shareable", "@provides", "@external"])
+  @link(url: "https://specs.apollo.dev/federation/v2.0",
+        import: ["@key", "@shareable"])
 ```
 
-This definition identifies a schema as a true Federation 2 schema. Without this definition, the composition process assumes a subgraph schema is a Federation 1 schema, and it applies intelligent defaults to support backward compatibility.
+This definition identifies a schema as a true Federation 2 schema, and it `import`s any federation-specific directives that the schema uses.
+
+> **Note:** Most subgraph schemas use the `@key` and `@shareable` directives, so we've included them in the snippet above. Depending on your schema, you might need to add other federated directives to the `import` array, such as:
+>
+> * `@inaccessible`
+> * `@external`
+> * `@provides`
+> * `@requires`
```

Comment:
Nit: We might want to add `@override` and `@tag`? Otherwise, if we'd rather avoid having to maintain/update that list so it includes all directives, then I think we might want to avoid having a list at all, and instead just link to the specific page listing them all. But as is, I worry this might confuse some (and to be clear, I do understand the phrasing doesn't actually pretend that this list is complete, but I still suspect some will read this a bit too fast to get the nuance).

Reply:
Cool, I've trimmed this to only include a couple directives that are also in Fed 1 so they look familiar, and make it seem less like we're telling folks exactly what to add.